shipstation_integration/items.py

```diff
@@ -1,7 +1,21 @@
 import frappe
 
 
-def create_item(product, store):
+def create_item(product, settings=None, store=None):
```

Comment:
We can't allow settings to be `None` as this is where we get the default item group from, and without the default Item group the mandatory check for Item will trigger.

Reply:
Ah, yeah. The calling functions always pass `settings`. I'll change the structure to reflect that.